pkg/cmd/server/server.go

```diff
@@ -60,15 +61,21 @@ import (
 )
 
 func NewCommand() *cobra.Command {
-	var kubeconfig string
+	var (
+		kubeconfig   string
+		logLevelFlag = newLogLevelFlag()
+	)
 
 	var command = &cobra.Command{
 		Use:   "server",
 		Short: "Run the ark server",
 		Long:  "Run the ark server",
 		Run: func(c *cobra.Command, args []string) {
-			logger := logrus.New()
-			logger.Hooks.Add(&logging.ErrorLocationHook{})
+			logLevel, err := logrus.ParseLevel(logLevelFlag.String())
+			if err != nil {
+				logLevel = logrus.InfoLevel
```

Comment:
Should we log (funny enough...) that the log level is invalid? Should it be an error?

Reply:
I had the same thought. Probably a good idea.

Reply:
Will do, although logically this case should never happen, since the enum flag will not allow an invalid value to be set.